x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/DataFrameMessages.java

```diff
@@ -26,7 +26,8 @@
     public static final String REST_PUT_DATA_FRAME_DEST_SINGLE_INDEX = "Destination index [{0}] should refer to a single index";
     public static final String REST_PUT_DATA_FRAME_INCONSISTENT_ID =
             "Inconsistent id; ''{0}'' specified in the body differs from ''{1}'' specified as a URL argument";
-
+    public static final String REST_UPDATE_DATA_FRAME_TRANSFORM_BATCH =
+        "Transform with id [{0}] cannot be updated. Updating is only supported on continuous transforms";
```

Comment:
whats the reason for this limitation?

Reply:
Colored in the details in overall comment response.